tappsbt/create.go

```diff
@@ -144,3 +147,63 @@ func OwnershipProofPacket(ownedAsset *asset.Asset,
 
 	return vPkt
 }
+
+// FromProofs creates a packet from the given proofs that adds them as inputs to
+// the packet.
+func FromProofs(proofs []*proof.Proof,
+	params *address.ChainParams) (*VPacket, error) {
+
+	pkt := &VPacket{
+		ChainParams: params,
+		Version:     0,
+	}
+
+	for idx := range proofs {
```

Comment:
We could spin the body of this loop into a `NewVInputFromProof` function. `FromProofs` would then be simpler and we might be able to reuse `NewVInputFromProof`.

Reply:
I see you've already included this commit in your PR (#896). I'd say let's do it when we actually need it (IMO because of the call to `pkt.SetInputAsset()` creating a `VInput` outside of the context of a `VPacket` would leave us with an incomplete `VInput`).